dev/CommonStyles/Common_themeresources_any.xaml

```diff
@@ -92,6 +92,11 @@
             <Color x:Key="SolidBackgroundFillColorQuarternary">#2C2C2C</Color>
             <Color x:Key="SolidBackgroundFillColorTransparent">#00202020</Color>
 
+            <Color x:Key="TabFillColorActiveLayer">#733A3A3A</Color>
```

Comment:
Why are these resources being defined in the common_themeresources file and note the tabview file?

Reply:
Should they be in the TabView_themeresources file? I figured all colours should be defined in this file, even if it's tabView specific.

Reply:
I think so, if they are tabview specific (which their name implies) they should be in the tabview file.